source/_integrations/androidtv_remote.markdown

```diff
@@ -484,3 +485,14 @@ cards:
 ```
 
 {% enddetails %}
+
+
+## Troubleshooting/Known issues
+
+- Doesn't work with Fire TV devices because they are missing the [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service). Attempts to sideload it haven't been successful.
```

Comment:
```suggestion
- The integration doesn't work with Fire TV devices because they are missing the [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service). Attempts to sideload it haven't been successful.
```